R/check-query.R

```diff
@@ -42,11 +43,13 @@ check_query <- function(query, endpoint) {
   fltxt_opr <- c("_text_all", "_text_any", "_text_phrase")
   all_opr <- c(simp_opr, num_opr, str_opr, fltxt_opr)
 
-  flds_flt <- fieldsdf[fieldsdf$endpoint == endpoint & fieldsdf$can_query == "y", ]
+  flds_flt <- fieldsdf[fieldsdf$endpoint == endpoint, ]
 
   apply_checks <- function(x, endpoint) {
     x <- swap_null_nms(x)
-    if (names(x) %in% c("_not", "_and", "_or") || is.na(names(x))) {
+
+    # troublesome next line:  'length(x) = 2 > 1' in coercion to 'logical(1)'
+    if (length(names(x)) > 1 || names(x) %in% c("_not", "_and", "_or") || is.na(names(x))) {
```

Comment:
The original line threw errors when checking queries.   Wasn't sure if this was the proper fix.